ui/src/app/app-routing.tsx

```diff
@@ -63,6 +65,8 @@ const CohortReviewPage = withRouteData(CohortReview);
 const ConceptHomepagePage = withRouteData(ConceptHomepage);
 const ConceptSetActionsPage = withRouteData(ConceptSetActions);
 const CookiePolicyPage = withRouteData(CookiePolicy);
+const DataPagePage = withRouteData(DataPage); // so called because 'WorkspaceData' is a swagger model
```

Comment:
PLEASE help me come up with better names for these components. The things in the comments are what I would use did they not exist already. Perhaps `WorkspaceDataComponent` -> `WorkspaceDataComponentPage`? `DataSetComponent` -> `DataSetComponentPage`??

Reply:
Yeah, I'm good with any of those. Maybe `DataComponentPage` and `DataSetComponentPage` makes the most sense, just replace the 'Page' from the component name for each with 'Component'?